pandas/io/formats/style_render.py

```diff
@@ -1187,3 +1207,38 @@ def _parse_latex_options_strip(value: str | int | float, arg: str) -> str:
     For example: 'red /* --wrap */  ' --> 'red'
     """
     return str(value).replace(arg, "").replace("/*", "").replace("*/", "").strip()
+
+
+def _escape_latex(s):
```

Comment:
where is eacape_html?

Reply:
imported from markupsafe which is a jinja2 dependency

Reply:
it is a very simple function, can write here instead of having dependency to markupsafe if preferred?

Reply:
oh its fine, just not completely obvious *where* it is (but its already there so its fine)